luigi/configuration.py

```diff
@@ -40,7 +40,7 @@
 class LuigiConfigParser(ConfigParser):
     NO_DEFAULT = object()
     _instance = None
-    _config_paths = ['/etc/luigi/client.cfg', 'client.cfg']
+    _config_paths = ['/etc/luigi/client.cfg', 'luigi.cfg', 'client.cfg']
```

Comment:
might be good if we can also show a warning on the cmd line that the location has moved